{
  "test_type": "unit",
  "timestamp": "2025-07-27T19:33:05.348546",
  "bug_id": "mrr_logic_errors_0641",
  "results": {
    "total": 63,
    "passed": 49,
    "failed": 5,
    "skipped": 4
  },
  "failures": [
    {
      "test": "test_logic_errors_0",
      "error": "Incorrect results",
      "file": "src/core/handler_54.py",
      "line": 166
    },
    {
      "test": "test_logic_errors_1",
      "error": "Precedence issues",
      "file": "src/core/handler_54.py",
      "line": 155
    }
  ]
}